### PR TITLE
[atip][XWALK-3539] Update tizen.py to get user id dynamically

### PR DESCRIPTION
**Pull request by chenxix**
- Tizen user can be given by param of lite dynamically.
- The default tizen user is app.

### CODE REVIEW

File: tools/atip/atip/tizen/tizen.py [hunk 1]
```diff
@@ -52,13 +52,15 @@ def do_cmd(cmd):
 
     return (cmd_return_code, output)
 
-def get_user_id(mode,device):
+def get_user_id(mode,device,tizen_user=None):
```
Comment (Honry) [inline tools/atip/atip/tizen/tizen.py]: you didn't update the place where calling this function

**Comment by Honry** [comment]
LGTM, @cicili please help double check. :)

**Comment by zqzhang** [comment]
@Honry @cicili can this pull request be landed? It affects the multi-user tasks.

**Comment by cicili** [comment]
@chenxix @Honry please add the UT result

**Comment by chenxix** [comment]
Verified on latest testkit-lite(v3.1.13) build, the userID can be achieved sucessfully.

**Comment by cicili** [comment]
thanks, and LGTM

**Pull request merged**